Added option for importing the private key (#37)
rarimo/rarime: packages/connector/README.md

```
Returns DID.

> ### get identity

Reviewer: ```suggestion
### Get identity
```

> ### get identity

Reviewer: let's add docs for createIdentity with pkHex

```
Returns DID.

### Get identity

> Returns DID and DID in big int string format if they are exists, otherwise method throws error.

Reviewer 2: ```suggestion
Returns DID in string and BigInt string formats. Throws error if the DID wasn't initialised yet.
```
